ui/src/main/resources/FileManagerCode/Macros.xml

```diff
@@ -505,6 +504,7 @@
         'user': "$!jobStatus.request.getProperty('user.reference')",
         'paths': $pathsAsJSON
       },
+      'error': "#if($jobStatus.error) $jobStatus.error #end",
```

Comment:
You need quotes if it is not `null` as the error is a word or even a sentence.
If `null` (no error happened), then I would be checking against the `"null"` string which is rather inelegant. 

Instead, the JS code checks with `if ( status.error )` which fires if `$jobStatus.error` was not null.

Reply:
I believe this answers the comment. Right? thanks to resolve.

Reply:
> If null (no error happened), then I would be checking against the "null" string which is rather inelegant.

Why? What do you get for:

    $jsontool.serialize({'key': $NULL})

because I get:

    {"key":null}

where ``null`` is not a string.

Reply:
Excellent suggestion. 
`      'error': $jsontool.serialize($jobStatus.error),`
is what I needed.
Thanks
Paul